### ISSUE TITLE
Mock 5.4 writes build log (and possibly other logs) into root log

### ISSUE THREAD

**Issue by Korulag** [rpm-software-management/mock#1302]
### Short description of the problem
It appears that mock 5.4 is not closing root log after preparing the environment and continues to write there the whole build process log. It was not the case on 4.1, I didn't check other 5.* versions because updates brought 5.4.
### Output of `rpm -q mock`
```# rpm -q mock

mock-5.4-1.el8.noarch
```

### Steps to reproduce issue

1. Build any package from src.rpm:
```mock --configdir $config_dir --root $root --verbose --rebuild $src.rpm --resultdir $rpm_result --define ... --no-cleanup-after```
2. Check produced root log

On 4.1 the root logs ends like so:
```DEBUG backend.py:799:  Copying packages to result dir
DEBUG util.py:167:  kill orphans
DEBUG util.py:621:  child environment: None
```
And a bunch of `umount` comands. On 5.4 it just continues.
Even more, it looks like `installed_packages` log is also included.

**Comment by praiskup** [comment]
Indeed, while build.log contains the output, it is duplicated in root.log. Thank you for the report.

**Comment by praiskup** [comment]
The `--verbose` option seems to trigger this misbehavior.

**Comment by Korulag** [comment]
Yes, we enable verbose output when using mock in some cases. Thanks for investigation @praiskup !
